beluga_amcl/package.xml

```diff
@@ -12,29 +12,41 @@
 
   <license>Apache License 2.0</license>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake</buildtool_depend>
 
   <depend>beluga</depend>
   <depend>bondcpp</depend>
```

Comment:
Is bondcpp useful in ROS 1? In ROS 2 we have nav2 lifecycle manager to monitor heartbeat, but I don't know about ROS 1.

Reply:
There used to be some managers out there (like http://wiki.ros.org/rocon_app_manager). Folks may have implemented their own too.

Reply:
New feature, you are not missing anything. It was pure inertia. It can be removed.

Reply:
I'd be inclined to remove it from the ROS 1 version, but it's your call.

Reply:
Removed in https://github.com/Ekumen-OS/beluga/pull/218/commits/8b05e2e7404f692fb5ba53f91987136f92e23b83.